### ISSUE TITLE
pnpm i -g X removing recently installed global packages

### ISSUE THREAD

**Issue by vjpr** [pnpm/pnpm#808]
```
> ls /Users/Vaughan/nvm/versions/node/v8.1.1/pnpm-global/node_modules/pnpm-global-pkg/node_modules
configurize             esformatter-preset-live live-frontend           live-stack-format       node-idea               shipmate
esformatter             live                    live-koa-server         live-static             pnpm-global-pkg         webpackerator
> pnpm i -g prettier
  prettier                                                          1.4.4 ✓
> ls /Users/Vaughan/nvm/versions/node/v8.1.1/pnpm-global/node_modules/pnpm-global-pkg/node_modules
configurize             live                    live-koa-server         live-static             pnpm-global-pkg         shipmate
esformatter-preset-live live-frontend           live-stack-format       node-idea               prettier                webpackerator
```

Notice `esformatter` is gone...

If they are installed together `pnpm i -g esformatter prettier`, then they are both installed.

### Additional information:

 - `node -v` prints: 8.1.1
 - Windows, OS X, or Linux?: macOS

**Comment by zkochan** [comment]
Was the `esformatter` installed with pnpm pre 0.70?

**Comment by vjpr** [comment]
Yeh it might have been.

**Comment by vjpr** [comment]
I just removed them completely and installed again, same issue. With 0.70.0 and .1.

**Comment by zkochan** [comment]
Fixed in [v0.72.0](https://github.com/pnpm/pnpm/releases/tag/v0.72.0)
